README.md

```diff
@@ -52,9 +52,11 @@ Experiments are in the `influence_benchmark/experiments` folder and have a large
 
 Custom environments can be defined as yaml files, see `influence_benchmark/config` for examples of this.
 
-An example command to run on a machine with a GPU looks like:
+An example command to run on a machine with a GPU looks like the commands below. We need at least two GPUs for expert iteration (EI), but one is sufficient for KTO, which is the preferred method. 
 
-`python influence_benchmark/experiments/run_experiment.py --config EI_test_up.yaml  --gpus 0`
+`python influence_benchmark/experiments/run_experiment.py --config KTO_up.yaml  --gpus 0`
+
+`python influence_benchmark/experiments/run_experiment.py --config EI_test.yaml  --gpus 2,4`
```

Comment:
Did you verify that both these worked in the end? Maybe we could remove the gpus flag, given that we'll likely use the auto-gpu code from now on? Maybe we could have a num_gpus flag and it will auto-figure them out